story03 (#4)
alem-platform/bootcamp-js: story03/mirror-issue/README.md

# The Mirror Issue

### Instructions

> Write a function called `mirrorIssue` where a local variable with the same name as a global variable, `hero`, shadows the global one. Inside `mirrorIssue`, the local `hero` should replace the text "global" with "local" and return the modified value, showing the "power struggle" between the global and local scopes. Do not pass the global variable directly into the function.

Reviewer: ```suggestion
Write a function called `mirrorIssue` where a local variable with the same name as a global variable, `hero`, shadows the global one. Inside `mirrorIssue`, the local `hero` should replace the text "global" with "local" and return the modified value, showing the "power struggle" between the global and local scopes.
```

> - In `mirrorIssue`, declare a local variable named `hero` that shadows the global one.

Reviewer: ```suggestion
- In `mirrorIssue`, declare a local variable named `hero`
```

> - Modify the local `hero` variable, changing "global" to "local", and return it.

Reviewer: ```suggestion
- Modify the local `hero` variable, by replacing the text "global" with "local" and return the modified value.
```

### Starting Code:

```js

> var mirrorIssue = function () {
>   var hero = "";
>   // ...
>   return hero;
> };

Reviewer: ```suggestion
var hero = "...";
var mirrorIssue = function () {
  var hero;
  // ...
  return hero;
};
```
